front/src/views/Users.vue

```diff
@@ -9,12 +9,12 @@
     <div class="modal" :class="{ 'is-active': showUserAdd }">
       <div class="modal-background"></div>
       <div class="modal-card">
+        <form @submit.prevent="addUser()" class="box">
```

Comment:
that's weird, in this file you'd overwrite my last commit.
maybe fetch and merge with my `main` branch ?

Reply:
You're right, I didn't see your last commit